### ISSUE TITLE
Add _rg.ps1 to $Env:PATH

### ISSUE THREAD

**Issue by jfishe** [dstcruz/chocolatey-mypackages#4]
Is it possible to put _rg.ps1 in the PATH like rg.exe?

It would be nice to enable completion without having to dot source the full path to `C:\ProgramData\chocolatey\lib\ripgrep\tools\ripgrep-12.1.1-x86_64-pc-windows-msvc\complete\_rg.ps1` which changes with the rg version.

Thanks for the package! Very useful.

**Comment by mattcargile** [comment]
Yes, I agree somewhat. I'd rather the folder where the files are be standardized. For instance, don't unzip the files into the parent folder with the name of the zip archive. I have to use this more complex code to ensure I always get the latest `_rg.ps1`.

```powershell
# ripgrep (rg) PSReadline Argument Completer
# Output of inner CLI tool
<#
ripgrep 13.0.0 (rev af6b6c543b)
-SIMD -AVX (compiled)
+SIMD +AVX (runtime)
 #>
$rgVersion = ( ( ( ( & rg.exe --version ) -split $newLn)[0] )  -split ' ' )[1] 
$rgPath = ( Resolve-Path "$env:ChocolateyInstall\lib\ripgrep\tools\*$rgVersion*\complete\" )[0].Path
. $rgPath\_rg.ps1
```
